### PR TITLE
[IMP] l10n_dk: Rework COA

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Backport of https://github.com/odoo/odoo/pull/121186 adapted to the old CoA templates engine.

@malb-odoo @Levizar

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_dk/data/account_group.xml
Comment (FlorianGilbert) [inline addons/l10n_dk/data/account_group.xml:1]: Could you please indent properly your file?
Please, translate the name in English and add `.pot` and `.po` file to translate it in Danish.
Why didn't you let the file in `.csv`?

File: addons/l10n_dk/data/account.account.template.csv
Comment (FlorianGilbert) [inline addons/l10n_dk/data/account.account.template.csv:1]: Ideally the file should be translated in English, but it seems the COA was already in Danish, then you could keep it like that.

File: addons/l10n_dk/data/l10n_dk_chart_template_post_data.xml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-        <field name="property_tax_receivable_account_id" ref="a8798"/>
+        <field name="property_account_receivable_id" ref="dk_coa_5960"/>
+        <field name="property_account_payable_id" ref="dk_coa_7180"/>
+        <field name="property_account_expense_categ_id" ref="dk_coa_1610"/>
+        <field name="property_account_income_categ_id" ref="dk_coa_1010"/>
+        <field name="property_account_expense_id" ref="dk_coa_1610"/>
+        <field name="property_account_income_id" ref="dk_coa_1010"/>
+<!--        <field name="property_stock_account_input_categ_id" ref="a8450"/>-->
+<!--        <field name="property_stock_account_output_categ_id" ref="a6670"/>-->
+<!--        <field name="property_stock_valuation_account_id" ref="a6530"/>-->
```
Comment (FlorianGilbert) [inline addons/l10n_dk/data/l10n_dk_chart_template_post_data.xml:12]: ?

**Review by FlorianGilbert — CHANGES_REQUESTED** [review]
Hello! 
Thank you for your contribution.
Some stuffs to say here:
- I think it should be fine to have all accounts in English and translate them in Danish by using .pot/.po file. It means that you have to add the `_post_init_hook` in the manifest and should ask the user to do a `-u`.
- If you don't translate accounts, don't take my remark about the groups and keep it in Danish. 
- Why didn't you use `.csv` file instead of `.xml` file for groups?

**Comment by gawa-odoo** [comment]
@FlorianGilbert l10n_multilang is needed to translate the accounts/groups/etc... 
=> it's not a current dependency, so he can't add it for 16.0

**Review by FlorianGilbert — COMMENTED** [review]
Hello
As we have made some changes on the CoA in version 17.0 and because we don't receive response from you. I will close this PR.
Thank you again for your contribution!

**Pull request closed without merging**